docs/configuration.md

```diff
@@ -6,15 +6,49 @@ This document describes how to configure your epoch node installed using a relea
 
 ### Peer-to-peer network
 
-In order for your node to join the testnet, you need to specify in the configuration file, how peers (on the Internet) can contact your node - specifically the TCP port (`sync` > `port` parameter).
+**It is very important that a node not only can connect to other nodes, but that can accepts incoming connections as well, the more peer connections (both inbound+outbound) the node have the better will be it's overall p2p network latency (i.e. block propagation time).**
```

Comment:
that it can accept

Reply:
the more peer connections (...) the node has the better it's overall p2p network latency (...) will be.